aws/validators_test.go

```diff
@@ -2646,3 +2646,91 @@ func TestValidateLaunchTemplateId(t *testing.T) {
 		}
 	}
 }
+
+func TestValidateNeptuneParamGroupName(t *testing.T) {
+	cases := []struct {
+		Value    string
+		ErrCount int
+	}{
+		{
+			Value:    "tEsting123",
+			ErrCount: 1,
+		},
+		{
+			Value:    "testing123!",
+			ErrCount: 1,
+		},
+		{
+			Value:    "1testing123",
+			ErrCount: 1,
+		},
+		{
+			Value:    "testing--123",
+			ErrCount: 1,
+		},
+		{
+			Value:    "testing_123",
+			ErrCount: 1,
+		},
+		{
+			Value:    "testing123-",
+			ErrCount: 1,
+		},
+		{
+			Value:    randomString(256),
+			ErrCount: 1,
+		},
+	}
+
+	for _, tc := range cases {
+		_, errors := validateNeptuneParamGroupName(tc.Value, "aws_neptune_cluster_parameter_group_name")
+
+		if len(errors) != tc.ErrCount {
+			t.Fatalf("Expected the Neptune Parameter Group Name to trigger a validation error")
```

Comment:
One of the test cases is generating unit test failures:

```
--- FAIL: TestValidateNeptuneParamGroupNamePrefix (0.00s)
	validators_test.go:2733: Expected the Neptune Parameter Group Name to trigger a validation error
```

Can you please include `tc.Value` in the error messaging here and below, then fix the test case? Thanks!